packages/expect/src/spy_matchers.js

```diff
@@ -248,7 +248,7 @@ const formatReceivedCalls = (calls, limit, options) => {
     const count = calls.length - limit;
     const printedCalls = getPrintedCalls(calls, limit, ', ', printReceived);
     return (
-      `${but} it was ${options && options.isLast ? 'last ' : ''}called ` +
+      `${but} it was called ` +
```

Comment:
Is this intentional?

Reply:
yep, was not used. cc @pedrottimark just to be sure

Reply:
Yes, it looks like `option.isLast` has been obsolete since before the package was renamed as `expect`

Scary what several weeks away does to my reading knowledge of Jest code :(

Reply:
It was lost in #3913

Reply:
`git log -S isLast` only returned 3 commits, a nice way of finding where something was add/removed/changed, if you didn't know about it 🙂